modules/javafx.web/src/test/java/test/javafx/scene/web/CallbackTest.java

```diff
@@ -88,7 +87,6 @@ private void clear() {
         popupUi.clear();
     }
 
-    @Ignore("RT-34508")
     @Test public void testDefaultPopup() {
```

Comment:
This test, along with `testCustomPopup` is listed as unstable. Have you run it multiple times on different systems?

Reply:
Yes, I've run the test multiple times in all 3 platforms and it's passing everytime.

Reply:
OK. In that case, please close RT-34508, aka [JDK-8090082](https://bugs.openjdk.java.net/browse/JDK-8090082), as "Cannot reproduce".